### PR TITLE
Hide restart button on darp6 and galp4

### PR DESCRIPTION
**Pull request by mmstick**
This branch will be re-uploaded as `master_bionic` on approval.

### CODE REVIEW

**Comment by brs17** [comment]
@jackpot51 , @mmstick when attempting to reproduce https://github.com/system76/firmware-open/issues/56, the issue still persists on shutdown when the system is connected to AC power throughout.

**Comment by brs17** [comment]
Might need to change the label on the success page not not mention restarting when we are only showing the shutdown button https://github.com/pop-os/installer/blob/d02a38c18db2eda6109c02f7f4c4ee9e806f6498/src/Views/SuccessView.vala#L42

**Comment by mmstick** [comment]
So we tell them to turn the computer off and on again? :P

File: src/Views/SuccessView.vala [hunk 1]
```diff
@@ -39,7 +39,7 @@ public class SuccessView : AbstractInstallerView {
         title_label.xalign = 0;
         title_label.get_style_context ().add_class ("h2");
 
-        var description_label = new Gtk.Label (_("After restarting you can set up a new user, or you can shut down now and set up a new user later."));
+        var description_label = new Gtk.Label (_("Shut down now and create a user account on the next start."));
```
Comment (brs17) [inline src/Views/SuccessView.vala]: :+1:

**Pull request closed without merging**